controllers/aggregated_clusteroperator_controller.go

```diff
@@ -100,6 +101,6 @@ func (a *AggregatedClusterOperatorReconciler) SetupWithManager(mgr ctrl.Manager)
 		For(&openshiftconfigv1.ClusterOperator{}, builder.WithPredicates(predicate.NewPredicateFuncs(func(object client.Object) bool {
 			return object.GetName() == aggregateCOName
 		}))).
-		Watches(&source.Kind{Type: &platformv1alpha1.PlatformOperator{}}, handler.EnqueueRequestsFromMapFunc(util.RequeueBundleDeployment(mgr.GetClient()))).
```

Comment:
This was needed as I saw a panic in the aggregate CO controller when deploying these changes to a crc cluster. Updated the logic to instead requeue the aggregate CO resource during PO watch events.